readme.md

```diff
@@ -55,9 +55,13 @@ $ npm install chalk
 ## Usage
 
 ```js
+// for stdout: console.log, console.info, console.debug
 const chalk = require('chalk');
+// for stderr: console.error, console.warn
+const cherr = require('chalk/stderr');
 
 console.log(chalk.blue('Hello world!'));
+console.error(cherr.red('Oh, colored error!'));
```

Comment:
I don't want `stderr` examples here. It's not something most would need. You can add one example to the `chalk/stderr` section instead.

Reply:
IMHO this is place where requirement for separate this two is most visible.
At most I can split this to two snippets or add notice under original

Reply:
Reverted to previous with added notice